images/concept-rtc-cloud.drawio

```diff
@@ -0,0 +1 @@
+<mxfile host="Electron" modified="2021-08-17T06:06:55.722Z" agent="5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) draw.io/14.9.6 Chrome/89.0.4389.128 Electron/12.0.16 Safari/537.36" etag="4aHhq-pxoOtwSaHwH900" version="14.9.6" type="device"><diagram id="-Rux2A9cZbcUw8DSModv" name="Page-1">7Vtbc5s4FP41nmkf7OGOeUycy3a7bbNNZtrmZUcGGWsMyEXybX/9HiHAIHBip8bJppnJTODogvSdT+ciyT1zFK+vUzSffqIBjnqGFqx75kXPMHRHs+GfkGykxDBdTUrClAR5ra3glvyLc2FRbUECzGoVOaURJ/O60KdJgn1ek6E0pat6tQmN6l+doxA3BLc+iprSbyTgUykdGu5W/gcm4bT4su54siRGReV8JmyKArqqiMzLnjlKKeXyKV6PcCTQK3CR7a52lJYDS3HC92ngff9k2Evj65fPsx//fOPatff3fV83ZDdLFC3yGeej5ZsCAugG0IaX89WUcHw7R74oWYHGQTblcQRvOjwiNpcqmJA1hq+eM57SGR7RiKZZV6Y20k1jVJYUiAIW5xOa8Fz9ugXvzenlM17ilON1RZRP9xrTGPN0A1VysllmDv1qqzlDy2XTitZKIcrZEpZ9bQGFhxzTQ/DVXyO+eak9zGEr8Nbt58Z7eES4m+j6gK7fDZa6Y9ewdK0md61WLO2usPQ6xXI3U4+BpubU0Ox7TTSdgq1VNE2rM0vQQPOOJJtPo0sQfr2DPrXLJOgvGE57hhPBcM7H4ikUT+/uVvR9A3wBBwFPdRaRMAERpwL0QvoXGuPohjLCCRWlY8o5jaFCpBT4ADN8FMxL3k8pqOiPLnhEElBg4WU1EIYpCghUviApCGVvCU0FwC1qBf83FyOP16GIFQYJ5iuazthg7j9IkA4Wm2Mqhstt0kNvW2xuV+xouuED2PElwW/sOKJbU8gxfG5ymA1ySErc4nQpCPFKNc/k9E6sfdOoq3+oNdXvtWhf7yymsRrqv0z8dDPPkM0Mw5wSmG/FMFTo4PxciAwjg6rPMqzOoIJuzdcZYEV5aU0kq16vQXkeWtm6O6hHePZz88pu8Arw5mKKxtUF9RcxPhqp3ixVF5aq2GoogtyWKOa0jHIajPrz292bifqf8MlomKiWHLQzQo2u7j4kjhG7zj2Kk/tl/Hn1vW80bdTdFIMASw+IA/EcEE5TuQcozZeGElHAs6qSgu/GC1GQUF4W4KoXneHNe9EwxRmMjImu1Uh7DArCWFQXkTjLekrpIpyWfcrATOpvAE93DalUCZkQzJQhohCRhG3Hd7MYR8SHh494wzJ0GfUJkpNeEb79KiAn9DiK6EKUnd18kK0aCwQnwZnYFBWcjqA/4j89TVcZyDhKebP3THxFouITO4nK6CL18eM5EvQXYv54uIyD2s7uTtprA8fRhjXe60ZzI8tuIX4hS3GEOFnWt4zbFkM+iBtpDEtD7ikZh6GsJwlN3qq6xat0ZFn1jkw1JZHYPdBRUZFOJgzX6mTrt0TvF5Z000m8Lpqae9JU35Omp+Ggo7kDy3Nsx9UdS3eH9dDC8Z7IyKFmDwzbdR1d9yzN8Orbcp6627aDn0fjnnta7gnd3uZdiyiAhjRB0eVWeu4vwCcEedd7+/RHOViclD3GQftFcfDZ7SAoHm0q1bKgle0e8FA5ALHz486dq0Gp368f2cGDHMFRSW8+tPOc4p8LzDgrYyaIWzFokDWDHySIB/FwFn7IiGWShV0yBpF7k10upgRw+C4CbYhS89cf2avmFu8X6zwQl2+b6tsNRF2ApYirM+ETHQNM6ShuwtlziRqHRTN9bWC4w3oYX57yPs8yNtUTqicvYzXPeDScWbdOobvgxjRaHIxcSGyOkl5b1utL9ouMNw3H72CK8CfyhcqTSE0ytmlZjjxBMYk2sg10hOJ5VmiagsdTHC2xSHgbJfVOlES7XiZHKQohfY1RVC9e5aQR5ZYcZ1YYYQ7rqw9T9UkStraHXJT387xaFOepda2YJEG2nES5VhlaVshTlDAwPHHRfYLLCpA9B/WvV5uPkT8LIV1Lgr6CuWHbJdbK8xb5gLB5hHLUSSKS/y0qEUW8OiB1y6I7iysbAVslwWTLjg1x1QLuaTZPH9s4hxlOsJuaqZwan9ZuGo4+8MRGseYZnml5Vm0wuhor72tFTfWWhvnykkKzecbxOgPzffcwXnZgfnKPfmhg3neUwFx/ODBX658mMG9ubj4aLPya+3+SV9Iyt9LuZR7fzj/aWE8Mjbo5rPjeQDkfO6X/febExeomcdEGuqm7dZfXPH54zUbupA63bRf2LVF5S1S6dQlvFOqaQge7zTeV/Aar+reNZg6JPupnwO4pYw/b8wae5+iuMTQ9W3OMehz01EjE8YzB0LQd19N0w7Rs5XbHwafD5b5IvV2HYUrzwK68o6YR4YnYYhwTLu9EZIxmCZqzKZV+K7vNwThNZblg/waw61XvirD2u2litT/E6myhAX2BhIqocWtIvZYUkyDINhnafjSRWYhs50EsbjbD3J/22k8sxHvbItu5Gg64C6TeLdOLs7jqb3vaLgOpZ3ZPuQvkBtdL/EGn0ezj/ez6DKWfP/stt6BFrNIrr728SP1lKssHZdVVaGx3khp6adHeblUp6YbXcrG0rHMSTTU3837IRSfv7bHfWFfKpm7Lz5KOpSp43f62VNrj7U90zcv/AA==</diagram></mxfile>
```

Comment:
Should this file be here? I don't believe so, as this would end up being deployed on the site and be publicly accesible.

Reply:
Fixed by https://github.com/tinymce/tinymce-docs/pull/2091/commits/b3091a1ad3845121811a20d729af9663174b26ec